src/reporter/tests/utils.py

```diff
@@ -106,6 +107,15 @@ def send_notifications(service, notifications, service_path='/'):
     for n in notifications:
         r = requests.post(notify_url(), data=json.dumps(n), headers=h)
         assert r.ok
+    return r
+
+
+def insert_entities(entities: Union[List[dict], dict],
```

Comment:
the type hint says you could pass in either a list of entities or a single entity dict, but I think if you try passing in a single entity dict the function will bomb out? is that why you have that entity to list conversion here

- https://github.com/orchestracities/ngsi-timeseries-api/pull/668/files#diff-9f73b00050ccee8fc0acb7f18c5295ff93da919a3e927ac7969a9b88aedfb62eR188

?

Reply:
this wouldn't be a problem if we kept the `insert_entites` you deleted?